File: SchedulingFront-OLD/src/app/models/organizationUnit.ts
import { Company } from '../common/models/company';
import { BasePaging } from '../common/models/basePaging';
import { SelectListModel } from '../common/models/selectListModel';

export class OrganizationUnit {
    id: number;
    organizationUnitId: number;
    companyId: number;
    company: Company;
    code: string;
    name: string;
    active: boolean;
    bindScheduleToEmployee: boolean;
}

export class OrganizationUnitPaging extends BasePaging {
    code: string;
    name: string;
}

export class OrganizationUnitSelectListModel extends SelectListModel {
    bindScheduleToEmployee: boolean;

    constructor(id: number, name: string, bindScheduleToEmployee: boolean) {
        super(id, name);
        this.bindScheduleToEmployee = bindScheduleToEmployee;
    }
}

File: SchedulingAPI/Scheduling.Contract/Models/OrganizationUnit.cs
using CodebookManagement.Models;
using CompanyManagement.Models;

namespace Scheduling.Contract.Models
{
    public class OrganizationUnit
    {
        public int OrganizationUnitId { get; set; }
        public int CompanyId { get; set; }
        public Company Company { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public bool Active { get; set; }
        public bool BindScheduleToEmployee { get; set; }
    }
    public class OrganizationUnitPaging : CodebookPaging
    {
        public long OrganizationUnitId { get; set; }
    }
}